app/user-data-download/server/cronProcessDownloads.js

```diff
@@ -451,6 +454,22 @@ const generateUserAvatarFile = function(exportOperation, userData) {
 	}
 };
 
+function spliceByFileSize(fileList, size) {
+	let fileCounter = 0;
+	let sizeCounter = 0;
+
+	for (let i = 0; i < fileList.length; i++) {
```

Comment:
In here, is there a reason to not to use a map/for-of here? (the splice would early return the map too)

Reply:
not really. i can make it into a map